javascript-loops/main.js

```diff
@@ -0,0 +1,79 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  for (var i = 1; i < 11; i++) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  for (var i = 2; i < 21; i += 2) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  var count = 1;
+  var repeated = '';
+  for (var i = count; i <= times; i++) {
+    repeated += word;
+    count += 1;
+  }
+  return repeated;
+}
+
+function logEachCharacter(string) {
+  for (var i = 0; i < string.length; i++) {
+    console.log(i, string.charAt(i));
```

Comment:
Strings have some array like features, one of them is you can access specific characters with `[ ]`. so `string[i]` would also work here.